build/stf-run-ci/tasks/main.yml

```diff
@@ -85,11 +87,11 @@
   - name: Create base build list
     set_fact:
       build_list:
-        - { name: service-telemetry-operator, dockerfile_path: build/Dockerfile, image_reference_name: sto_image_path, working_build_dir: ../ }
-        - { name: smart-gateway-operator, dockerfile_path: build/Dockerfile, image_reference_name: sgo_image_path, working_build_dir: ./working/smart-gateway-operator }
-        - { name: sg-core, dockerfile_path: build/Dockerfile, image_reference_name: sg_core_image_path, working_build_dir: ./working/sg-core }
-        - { name: sg-bridge, dockerfile_path: build/Dockerfile, image_reference_name: sg_bridge_image_path, working_build_dir: ./working/sg-bridge }
-        - { name: prometheus-webhook-snmp, dockerfile_path: Dockerfile, image_reference_name: prometheus_webhook_snmp_image_path, working_build_dir: ./working/prometheus-webhook-snmp }
+        - {name: service-telemetry-operator, dockerfile_path: build/Dockerfile, image_reference_name: sto_image_path, working_build_dir: "{{ base_dir }}/../"}
```

Comment:
This looks suspicious to me, wondering if the whole value should be quoted. I think the Jenkins testing will catch this?

Reply:
It has to be quoted since it's using a jinja2 template to render the var in the name

Reply:
I was wondering whether the entire line would need the quotes, rather than just the working_build_dir property. Seems not. :)